renderdoc/os/posix/posix_process.cpp

```diff
@@ -626,8 +628,8 @@ static pid_t RunProcess(rdcstr appName, rdcstr workDir, const rdcstr &cmdLine, c
       }
 
       chdir(workDir.c_str());
-      execve(appPath.c_str(), argv, envp);
-      fprintf(stderr, "exec failed\n");
+      RD_EXECVE(appPath.c_str(), argv, envp);
```

Comment:
extreme nit: can you change this to `RD_EXECVPE` since that's what we want to use, it's just not available on OSX.